### PR TITLE
Update zebra-lifeguard-ota-integration.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated pre requisites with ports to be opened

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3902

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/protect/zebra-lifeguard-ota-integration.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ The following aren't supported in public preview:
   - Android FOTA (to manage firmware OTA updates)
 
 - Access to all appropriate Zebra licenses, and entitlements to use the LG OTA service. For more information, contact Zebra support or see [Zebra's TechDocs](https://techdocs.zebra.com/lifeguard/faq/).
+- For information about services ports and endpoints used by Zebra OTA updates, refer to [Zebra Lifeguard Over the Air FOTA Updates Ports](https://supportcommunity.zebra.com/s/article/000022419?language=en_US)
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/zebra-lifeguard-ota-integration.md:64]:
```suggestion
- For information about services ports and endpoints used by Zebra OTA updates, refer to [Zebra Lifeguard Over the Air FOTA Updates Ports](https://supportcommunity.zebra.com/s/article/000022419?language=en_US).
```

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@Smritib17 Please review and signoff this PR. Thanks

**Pull request merged**